### PR TITLE
FFT Solutions and Video

### PR DESCRIPTION
**Pull request by nwatx**

### CODE REVIEW

File: content/6_Advanced/FFT.mdx [hunk 1]
```diff
@@ -122,18 +124,240 @@ export const problems = {
...
+void unsyncIO() { cin.tie(0)->sync_with_stdio(0); }
+void setIO() {
+    unsyncIO(); setPrec();
+}
+
+#define rep(i, a, b) for(int i = a; i < (b); ++i)
+typedef pair<int, int> pii;
+
+typedef complex<double> C;
+void fft(vector<C>& a) {
```
Comment (bqi343) [inline content/6_Advanced/FFT.mdx:297]: cite the source of this?

File: content/6_Advanced/FFT.mdx [hunk 2]
```diff
@@ -17,17 +17,17 @@ export const problems = {
 			'convolution_mod',
 			'Easy',
 			false,
-			[],
-			''
+			['fft'],
```
Comment (bqi343) [inline content/6_Advanced/FFT.mdx:20]: Capitalize tag

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: content/6_Advanced/FFT.mdx [hunk 3]
```diff
@@ -17,17 +17,17 @@ export const problems = {
 			'convolution_mod',
 			'Easy',
 			false,
-			[],
-			''
+			['FFT'],
+			'convolution-mod'
```
Comment (bqi343) [inline content/6_Advanced/FFT.mdx:21]: why are there separate solution files as well as sols within the module? just put them in the module

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: content/6_Advanced/FFT.mdx (hunk 3 above)
Comment (nwatx) [reply content/6_Advanced/FFT.mdx:21]: it was like that before. i'll remove the internal sols then

**Review by bqi343 — APPROVED** [review]

**Pull request merged**